.ci/skipped_notebooks.yml

```diff
@@ -585,3 +585,8 @@
     - os:
         - macos-12
         - windows-2019
+- notebook: notebooks/whisper-asr-genai/whisper-asr-genai.ipynb
+  skips:
+    - python:
+        - '3.8'
```

Comment:
we dropped python3.8 support from notebooks, so it should not be required to add. What the issue with python3.9?